openshift-container-platform-4/policies/RA-Risk_Assessment/component.yaml

```diff
@@ -149,12 +149,22 @@
 - control_key: RA-5 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Progress can be tracked via:
+        Organizational policy and procedures to employ and update the organizational
+        scanning appliance is outside the scope of OpenShift configuration.
+        The Quay container registry that Red Hat recommends for container storage can be
+        integrated with the Clair vulnerability scanning tool
+        Documentation for adding a Clair container to perform scans:
+        https://access.redhat.com/documentation/en-us/red_hat_quay/2.9/html/manage_red_hat_quay/clair-initial-setup
+
+        Clair vulnerability scan data sources and drivers documentation:
+        https://github.com/quay/clair/blob/master/Documentation/drivers-and-data-sources.md
+
+        Documentation progress can be tracked at:
```

Comment:
```suggestion
        Organizational policies and procedures to employ an organizational
        scanning tool are outside the scope of OpenShift configuration.

        To aid OpenShift system administrators, the Quay container registry
        that Red Hat recommends for container storage can be
        integrated with the Clair vulnerability scanning tool.
        Documentation for adding a Clair container to perform scans can
        be found at:

        https://access.redhat.com/documentation/en-us/red_hat_quay/2.9/html/manage_red_hat_quay/clair-initial-setup

        Clair vulnerability scan data sources and drivers documentation:
        https://github.com/quay/clair/blob/master/Documentation/drivers-and-data-sources.md
```

Reply:
Minor copy editing.

Is there a non-github link for the clair docs? Something from docs.redhat.com or the customer portal?